src/components/Icons.jsx

```diff
@@ -152,19 +152,22 @@ import SmartFX from '../../sass/icons/active-symbols/ic-smartfx-placeholder.svg'
 
 const Wrapper = WrappedComponent => props => {
     let { className, ['tooltip-title']: tooltip, ...p } = props;
-    className = `ic-icon ${className ? className : ''} ${tooltip ? 'tooltip' : ''}`;
+    className = `ic-icon ${className ? className : ''}`;
 
     return (
         <span
             className={className}
             tooltip-title={tooltip}
```

Comment:
this is no longer used right? Should be safe to remove

Reply:
Since we are using Icon.jsx other places and some needs to show tooltip( you still wants some icons to show tooltip,right? for instance open draw-tools there are two icons that need tooltip), it should be remove from here but for those icons which still needs to show tooltips we should set this class while we're creating them that i will do if its the case.

Reply:
I see. Then where it is is fine

Reply:
No we have two types of icons, one with tooltip and one without. I added to tooltip class to those who needs it.